src/app/public/plugin-resources/code-examples/inline-form/basic/inline-form-demo.component.html

```diff
@@ -37,19 +37,21 @@
     <div
       class="sky-form-group"
     >
-      <label
-        class="sky-control-label"
-        [for]="firstNameInput.id"
-      >
-        First name
-      </label>
-      <input
-        class="sky-form-control"
-        formControlName="myFirstName"
-        skyId
-        type="text"
-        #firstNameInput="skyId"
-      />
+      <sky-input-box>
+        <label
+          class="sky-control-label"
+          [for]="firstNameInput.id"
+        >
+          First name
+        </label>
+        <input
+          class="sky-form-control"
+          formControlName="myFirstName"
+          skyId
+          type="text"
+          #firstNameInput="skyId"
+        />
+      </sky-input-box>
```

Comment:
for this and all of the following, all i did was surround the `<label>` and `<input>` with a `sky-input-box`